recipes/finetuning/datasets/raft_dataset.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed according to the terms of the Llama 3 Community License Agreement.
+
+
+import copy
+from datasets import load_dataset
+import itertools
+
+B_INST, E_INST = "[INST]", "[/INST]"
```

Comment:
do we still need this or apply_chat_template is just doing it?

Reply:
I should delete this, as it is actually for Llama 2, but I am not going to support Llama 2 as the prompts are already designed for Llama3.